packages/manager/apps/public-cloud/src/navbar/index.js

```diff
@@ -0,0 +1,52 @@
+import angular from 'angular';
+import resource from 'angular-resource';
+import translate from 'angular-translate';
+import _ from 'lodash';
```

Comment:
scoped imports

Reply:
Imports can't be scoped since the `"lodash": "^3.10.1"` has been defined in the dependencies' list.

`@ovh-ux/manager-layout-ovh` is already in that case.